### PR TITLE
[MVC-985] CMS- Linking uneditable features with CMS

### PR DESCRIPTION
**Pull request by nashthecoder**
[MVC-985] CMS- Linking uneditable features with CMS
- Removed commented code, 
- Added target=_blank on social icons to enable click on social icons to open on new window

[MVC-985]: https://triggerise.atlassian.net/browse/MVC-985

### CODE REVIEW

File: _includes/footer.html [hunk 1]
```diff
@@ -3,51 +3,12 @@
...
-              </svg>
-            </a>
-          </li>
-        </ul>
+          <div class="socials">
+            {% assign social = site.social-icons%}
+              {%for social in social%}
+                {{ social.content }}
+              {% endfor %}
+          </div>
```
Comment (franciscoovazevedo) [inline _includes/footer.html]: Looks great!

File: _sass/_base.scss [hunk 2]
```diff
@@ -364,33 +364,34 @@ footer {
...
       list-style: none;
-      margin: 0;
-
+      margin-left: 0;
       li {
         margin-right: 10px;
       }
     }
   }
+//Footer Icons
```
Comment (franciscoovazevedo) [inline _sass/_base.scss]: should we have an empty line here? just to keep it consistent with the rest of the file.

**Review by franciscoovazevedo — APPROVED** [review]
Great work, much easier to read now. 

Just left one small note, but great job on this!

File: _sass/_base.scss (hunk 2 above)
Comment (nashthecoder) [reply _sass/_base.scss]: @franciscoovazevedo Thanks for the feedback, will clean it up.

Comment (nashthecoder) [reply _sass/_base.scss]: @franciscoovazevedo done.

**Pull request merged**